src/OpenTelemetry.Exporter.Console/ConsoleExporterMetricsExtensions.cs

```diff
@@ -25,6 +26,9 @@ namespace OpenTelemetry.Metrics
     /// </summary>
     public static class ConsoleExporterMetricsExtensions
     {
+        private const int DefaultExportIntervalMilliseconds = Timeout.Infinite;
+        private const int DefaultExportTimeoutMilliseconds = 30000;
```

Comment:
Do we need timeout 30000 or Infinite?

Reply:
I just submitted https://github.com/open-telemetry/opentelemetry-specification/pull/2415 for further clarification.

I'll set to Infinite for now and change depending on how things shake out.